features/polyExplorer/src/constants/index.js

```diff
@@ -20,21 +20,3 @@ export const INTRO_ANIMATIONS = {
     FORWARD: "forward",
     BACKWARD: "backward",
 };
-
-export const SUMMARY_ANIMATIONS = {
-    ANIMATE_BULLET_ONE: "ONE",
-    ANIMETE_BULLET_TWO: "TWO",
-    ANIMATE_BULLET_THREE: "THREE",
-};
-
-export const DONUT_CHART = {
-    DEFAULT_GROUP: "default",
-    DEFAULT_COLOR: "#3BA6FF",
-    FACEBOOK_GROUP: "Facebook",
-    FACEBOOK_COLOR: "#3749A9",
-};
-
-export const BUBBLES_SPEECH_SIZES = {
```

Comment:
This constant (BUBBLES_SPEECH_SIZES) is needed at the moment. It's used for the `LinesChart` component in `details.jsx`.